.github/workflows/pythonapp.yml

```diff
@@ -9,10 +9,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Install poetry pylint
+    - name: Install system dependencies
+      run: sudo apt install gobject-introspection pkg-config libcairo-dev libgirepository1.0-dev libdbus-1-dev
+    - name: Install poetry
+      run: pipx install poetry==v1.6.1
```

Comment:
That pins poetry to 1.6.x right ? I don't mind this for CI but I guess we are still broken for users who has that version installed

Reply:
Yes it fix poetry version. 

I don't find why it's not working, but I'm not an expert with python an poetry.

I'm gonna ask

Reply:
ah if you find out that would be great!